src/domain/entities/note.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Note entity
+ */
+export default class NoteEntity {
```

Comment:
I'd suggest simply `Note` like we call it in real-world

Reply:
Done